apps/contrib/templates/a4_candy_contrib/item_detail.html

```diff
@@ -17,10 +17,19 @@
                 <nav class="breadcrumbs" aria-label="{% translate 'breadcrumbs' %}">
                     <ul>
                         <li>
-                            <a href="{{ module.get_detail_url }}">
+                          {% with navigate_url=request.META.HTTP_REFERER %}
+                            {% if "?" in navigate_url %}
```

Comment:
I think the condition here is not working as intended. From how I understand the story there are two cases:

1. you open a project and have list of ideas. You click on an idea to open the detail view. In this case we want to show the `go back` button which will take you back to the list with correct page/pagination and filters set
2. someone shares a link to an idea. In this case we want to show the `go to overview` button and just go back without pagination/filtering.

By checking for ? we actually show the `go back to overview` if you click on any idea on page > 1,  a non-default sorting or a filter is set.  Opening an idea from a link will show `go back`  instead of `go back to overview`.

Reply:
I actually understood it as I have implemented it. When we go to a detail view from the pagination we go back to the overview of the ideas, when we share the link we go back to the project page. I will look at the story again.

Reply:
So the `only when coming from link: go to overview` means when the link is shared, that wasn't clear. I will change it accordingly. The qn now is, should both cases navigate to the part of the page with the items list? so the `go to overview` should also take the user to the default ordering/filtered list? Or `go to overview` takes back at the header/title of the project, meaning the `{{navigate_url}}`without the `index` anchor?

Reply:
hmm, not sure, I think going to the index/list instead to the top of the page when coming from a link would be fine but we can also ask @lthueer what he prefers

Reply:
I will leave it now going back to the index anchor, we can always change it if that is not the case.

Reply:
@goapunk fixed. I now check for the module's url in the HTTP-referer, which is enough and safer to distinguish between the 2 cases.